docs/guides/development/integrations/databases/convex.mdx

```diff
@@ -164,6 +164,12 @@ With [Convex](https://www.convex.dev/), you can build a backend with a provided
         throw new Error('Add your Clerk Publishable Key to the .env file')
       }
 
+      if (!process.env.PUBLIC_CONVEX_URL) {
```

Comment:
This is an excellent catch! Thanks for the contribution, @manythingsapps.

Should we use `VITE_CONVEX_URL` instead? That's what the Convex [docs](https://docs.convex.dev/auth/clerk#react) and [template](https://github.com/get-convex/template-react-vite-clerk/blob/main/src/main.tsx#L10) both use.